run.py

```diff
@@ -764,6 +764,8 @@ def main():
                         help="Exit on test failure")
     parser.add_argument("--testdir", action="store",
                         help="Runs tests from custom directory")
+    parser.add_argument("--skip-tests", nargs="?", default=None,
+                        help="Runs tests from custom directory")
```

Comment:
The long description is copy-pasted 
It should be updated to `Skips tests with this pattern`

Reply:
Oops. Fixing this. Thanks.